src/types.ts

```diff
@@ -71,6 +72,7 @@ export type FilterType =
   | "overall_risk_of_bias"
   | "isotypes_reported"
   | "specimen_type"
+  | "antibody_target"
```

Comment:
just had a question about the requirement of this change.

Reply:
Get this error otherwise unfortunately: 
![image](https://user-images.githubusercontent.com/33983945/141535510-52ea0b42-b0a5-4c4c-a646-659fb4fcd1ae.png)